pkg/asset/agent/image/baseiso.go

```diff
@@ -110,7 +101,13 @@ func (i *BaseIso) Generate(dependencies asset.Parents) error {
 
 	var baseIsoFileName string
 	var err error
+	archName := manifests.X86CPUArchitecture
```

Comment:
It could be useful to higlight somehow that's the default arch, at least with one comment.

Reply:
@andfasano ended up using `DefaultCPUArchitecture` similar to AssistedInstaller.
https://github.com/openshift/assisted-service/blob/master/internal/common/common.go#L46